databricks/koalas/frame.py

```diff
@@ -1155,27 +1155,36 @@ def nunique(self, axis: int = 0, dropna: bool = True, approx: bool = False,
             Don’t include NaN in the count.
         approx: bool, default False
             If False, will use the exact algorithm and return the exact number of unique.
-            If True, it uses Spark's approximate algorithm, which is faster in most circumstances.
-            Note: this parameter is specific to Spark and is not found in pandas.
+            If True, it uses the HyperLogLog approximate algorithm, which is significantly faster
+            for large amount of data.
+            Note: this parameter is specific to Koalas and is not found in pandas.
```

Comment:
Minor note, but I guess it should be capitalized like `Note: This ...` (I know, this mistake is originally from my PR 🙈)